### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by mansaj**
PR template

### CODE REVIEW

**Review by flozia — APPROVED** [review]
Awesome!

**Comment by flodolo** [comment]
FYI, it's also possible to use forms instead of a markdown template. 
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

You can see an [example here](https://github.com/mozilla-l10n/nimbus-l10n/issues/new?assignees=flodolo&labels=&template=new_translation.yaml&title=%5BNEW%5D%3A+%28experiment+name%29) of how it looks like

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Screenshot (if applicable)
+
+Not applicable.
+
+# How to test
+
+
+
+# Checklist (Definition of Done)
+- [ ] l10n changes have been submitted to the l10n repository, if any.
```
Comment (toufali) [inline .github/pull_request_template.md:22]: For Relay, they have a separate repo/server I believe, and this line is more applicable to them.  For us, we might say something like below?
```suggestion
- [ ] Localization strings (if needed) have been added.
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
+- [ ] I've added or updated relevant docs in the docs/ directory.
```
Comment (toufali) [inline .github/pull_request_template.md:24]: We don't have a docs directory yet in /src – should we add this, or try to reuse the one in root?

**Review by toufali — APPROVED** [review]

File: .github/pull_request_template.md (hunk 2 above)
Comment (mansaj) [reply .github/pull_request_template.md:24]: Let me word it a little differently.. How about: 
`I've added or updated the relevant sections in readme and/or code comments`

Comment (mansaj) [reply .github/pull_request_template.md:24]:
```suggestion
- [ ] I've added or updated the relevant sections in readme and/or code comments
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<!-- The following is intended to be helpful to you. Feel free to remove anything that is not. -->
+
+# JIRA Ticket:
```
Comment (mansaj) [inline .github/pull_request_template.md:3]:
```suggestion
# References: 
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<!-- The following is intended to be helpful to you. Feel free to remove anything that is not. -->
+
+# JIRA Ticket: 
+MNTOR-
```
Comment (mansaj) [inline .github/pull_request_template.md:4]:
```suggestion
Jira: MNTOR-
Figma: 
```

**Pull request merged**